### DISCUSSION TITLE
Sqids for Bazel

### DISCUSSION THREAD

**Discussion by mgred** [sqids/sqids-spec#16] [Ideas]
Hey, I created a port of [Sqids for Bazel](https://github.com/mgred/sqids-bazel). It's mostly adopted from [sqids-python](https://github.com/sqids/sqids-python/) since starlark is a dialect of python and passes all test so far. I'm curious if we could move the repo to this organization and publish it from there.
What do you think? I'm thankful for any feedback and improvements in addition to it.

**Comment by 4kimov** [comment]
Hi @mgred,

Thank you for the heads up and the work! Yes, definitely let's do that 👍 
I've sent you an invite to [sqids-bazel](https://github.com/sqids/sqids-bazel), let's move our discussion here: https://github.com/sqids/sqids-bazel/issues/1

**Reply by mgred** [reply]
Nice, thanks :+1:
